newa/cli.py

```diff
@@ -291,8 +302,12 @@ def cmd_schedule(ctx: CLIContext) -> None:
 def cmd_execute(ctx: CLIContext, workers: int) -> None:
     ctx.enter_command('execute')
 
-    if not os.environ.get("TESTING_FARM_API_TOKEN"):
+    tf_token = ctx.settings.get("tf_token")
```

Comment:
Same as above, `ctx.settings.tf_token` should work and be verifiable by linters.

Reply:
Fixed.